src/components/Card/Card.js

```diff
@@ -16,9 +19,18 @@ export const Card = ({ items }) => {
           <i className="fas fa-heart"></i>
         </Link>
         <div className="card-header-txt">
-          <Link to="/single-product">
-            <h3 className="semibold t1p125">{items.title}</h3>
-          </Link>
+          <div className="flex-row-wrap align-items-center justify-sb">
+            <Link to="/single-product">
+              <h3 className="semibold t1p125">{items.title}</h3>
+            </Link>
+
+            <span className="rating-badge rating-badge-ecom">
+              3.5{" "}
+              <span className="rating-fill rating-fill-ecom">
```

Comment:
Remove this hardcoded rating.

Reply:
Oops! Thank you for pointing it out, will fix it right away!